### DISCUSSION TITLE
How to use "OrchardCore.Localization.Core" with po from url file without downloading?

### DISCUSSION THREAD

**Discussion by KhoiNguyenICT** [OrchardCMS/OrchardCore#7080] [Q&A]
I am using the package "OrchardCore.Localization.Core".
I want to upload the .po file to the storage server for use. How can I use the .po file on the storage server without downloading it?
Thanks!

**Comment by hishamco** [comment]
> How can I use the .po file on the storage server without downloading it?

Could you please elaborate what you mean by "without downloading it"?

If I understand you correctly you want to use localization package and the translation files will be somewhere else, so my question is there anything prevent you to store them in the same web server?

**Reply by KhoiNguyenICT** [reply]
I have a server that stores the configuration to store message components to translate into the database, then I will put those files when exported from the database into .po files. The configurations inside the .po file are all dynamic so I don't want to host them in the same web server

**Reply by hishamco** [reply]
As I see from your scenario you can either:

1- Create a `DatabaseStringLocalizer` to load the translation from within the database directly instead of long life cycle or

2- Implement `ILocalizationFileLocationProvider` to load the translation from other server

**Reply by KhoiNguyenICT** [reply]
I also thought about the possibility of using stream data from a static file from url, then the script pushed into the memory cache will be executed, when the translation lifecycle will continue the old way.
